files/cloudwatch-agent-config.json

```diff
@@ -0,0 +1,38 @@
+{
+  "agent": {
+    "debug": true
+  },
+  "logs":{
+    "metrics_collected":{
+      "prometheus":{
+        "log_group_name": "graphdb",
+        "prometheus_config_path": "/etc/prometheus/prometheus.yml",
+        "emf_processor": {
+          "metric_declaration_dedup": true,
+          "metric_namespace": "GraphDB-Metrics",
+          "metric_unit":{
+            "graphdb_class_count": "Count",
+            "graphdb_cpu_load": "Percent"
+          },
+          "metric_declaration":[
+            {
+              "source_labels": [
+                "job"
+              ],
+              "label_matcher": "graphdb_infrastructure_monitor",
+              "dimensions": [["host"]],
+              "metric_selectors":[
+                "^graphdb_class_count$",
+                "^graphdb_cpu_load$",
+                "^graphdb_work_dir_free$",
```

Comment:
what about data dir?

Reply:
Added graphdb_data_dir_free and graphdb_data_dir_used.